modules/ServerAPI/src/server/python/createLiveDesignLiveReportForACAS/create_lr_for_acas.py

```diff
@@ -153,29 +154,56 @@ def extract_endpoints_dict(node, endpoints_dict):
         extract_endpoints_dict(child_node, endpoints_dict)
     return endpoints_dict
 
+def wait_and_add_columns(ld_client,
+                         live_report_id,
+                         project_id,
+                         assay_name,
+                         column_names=None,
+                         max_retries=10,
+                         delay_sec=5):
+    """
+    Attempt to find and add columns with specified names into a LR.
+    If the columns don't exist, poll for `retries` times with `delay_sec` delay between tries
+    until either success or timeout.
+    :param ld_client: LD client
+    :param live_report_id: ID of LiveReport to add columns to
+    :param column_ids: Assay name of columns: if column is 'Foo (bar)' then assay_name = 'Foo'
+    :param column_names: Full column names of format 'Foo (bar)'
+    :param max_retries: Maximum of retries, defaults to 10
+    :param delay_sec: Seconds of delay been tries, defaults to 5
+    """
+    success = False
+    for attempt in range(max_retries):
+        try:
+            col_ids = find_column_ids(
+                ld_client, assay_name, project_id=project_id, column_names=column_names)
+            if not col_ids:
```

Comment:
I've seen issues where it takes longer for some data (I believe the dose response curves) to make their way to LD.  I think it would be better to do:
```
if len(col_ids) != len(col_names):
```